metagraph/src/cli/config/config.cpp

```diff
@@ -181,11 +181,11 @@ Config::Config(int argc, char *argv[]) {
         } else if (!strcmp(argv[i], "--max-count-q")) {
             max_count_quantile = std::min(std::stod(get_value(i++)), 1.);
         } else if (!strcmp(argv[i], "--count-bins-q")) {
-            for (const auto &border : utils::split_string(get_value(i++), " ")) {
+            for (const auto &border: utils::split_string(get_value(i++), " ")) {
```

Comment:
```suggestion
            for (const auto &border : utils::split_string(get_value(i++), " ")) {
```

Reply:
When I merge it already says its up to date. Maybe because I rebased the commits from master into the branch before?

Reply:
yeah, weird...